canonical/ubuntu.com
# Update kernel release diagram

## Done

Decided to do a complete replacement of the diagram. [Conversation in the Jira ticket](https://warthogs.atlassian.net/browse/WD-11813?focusedCommentId=501405) shows how it progressed, product manager and reporter notified.

## QA

- Go to https://ubuntu-com-13922.demos.haus/kernel/lifecycle#installation-20-04, check that new diagram matches https://ubuntu-com-13922.demos.haus/about/release-cycle#ubuntu-kernel-release-cycle
    - Be sure to test on mobile, tablet and desktop screen sizes

## Issue / Card

Fixes https://github.com/canonical/ubuntu.com/issues/13915
Fixes https://warthogs.atlassian.net/browse/WD-11813

## Screenshots

[If relevant, please include a screenshot.]


## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
